MovieGallery/Pages/Search.cshtml

```diff
@@ -105,63 +105,76 @@
+                        <th>Crew</th>
+                        <th>IMDb Rating</th>
+                        @*<th>image</th>*@
+                        <th>Type</th>
                     </tr>
-                }
-            </tbody>
-        </table>
-        <table class="table table-striped table-light">
-            <thead class="thead-dark">
-                <tr>
-                    <th>Title</th>
-                    <th>Crew</th>
-                    <th>IMDb Rating</th>
-                    @*<th>image</th>*@
-                    <th>Type</th>
-                </tr>
-            </thead>
-            <tbody>
-                @foreach (var show in Model.ShowsResult.Items)
-                {
+                </thead>
+                <tbody>
+                    @foreach (var movie in Model.MoviesResult.Items)
+                    {
 
+                        <tr>
+                            <td>@movie.FullTitle</td>
+                            <td>@movie.Crew</td>
+                            <td>@movie.ImDbRating</td>
+                            <td>@movie.Type</td>
+                            @*<td>@movie.Image</td>*@
+                        </tr>
+                    }
+                </tbody>
+            </table>
+            <table class="table table-striped table-light">
+                <thead class="thead-dark">
                     <tr>
-                        <td>@show.FullTitle</td>
-                        <td>@show.Crew</td>
-                        <td>@show.ImDbRating</td>
-                        <td>@show.Type</td>
-                        @*<td>@movie.Image</td>*@
+                        <th>Title</th>
+                        <th>Crew</th>
+                        <th>IMDb Rating</th>
+                        @*<th>image</th>*@
+                        <th>Type</th>
                     </tr>
-                }
-            </tbody>
-        </table>
+                </thead>
+                <tbody>
+                    @foreach (var show in Model.ShowsResult.Items)
+                    {
+
+                        <tr>
+                            <td>@show.FullTitle</td>
+                            <td>@show.Crew</td>
+                            <td>@show.ImDbRating</td>
+                            <td>@show.Type</td>
+                            @*<td>@movie.Image</td>*@
+                        </tr>
+                    }
+                </tbody>
+            </table>
+        }
+        else
+        {
+            <div class="row">
+                <div class="col">
+                    <div class="alert alert-info">
+                        No Data Present for this year
+                    </div>
+                </div>
+            </div>
+        }
     }
     else
     {
         <div class="row">
             <div class="col">
                 <div class="alert alert-info">
-                    No Data Present for this year
+                    Please enter a valid year!
```

Comment:
Message displayed to show that the user is entering the year in an incorrect format.

Reply:
Where will the user enter it?

I'd lean toward keeping the original message.